README.md

```diff
@@ -367,6 +367,21 @@ CK8S_APPS_PATH= # fill this in
 source <($CK8S_APPS_PATH/bin/ck8s completion bash)
 ```
 
+### Upgrading compliantkubernetes-apps
+
+The [`bin/ck8s`](bin/ck8s) script also provides commands to upgrade an environment in two steps `prepare` and `apply`.
+The former runs scripted configuration steps that may not change the state of the environment, while the latter runs scripted upgrade steps that modifies the state of the environment.
+On unexpected failures the command will try to perform a rollback when possible to ensure that the environment continues to function.
+
+```bash
+./bin/ck8s upgrade vX.Y prepare
+./bin/ck8s upgrade vX.Y apply
+```
+
+It is possible to upgrade from one minor version to the next regardless of patch releases.
```

Comment:
You could also mention that one can upgrade to a later patch in the same minor version as well. E.g. 0.x.y -> 0.x.z